docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -601,3 +601,24 @@ Which is the same type as the one accepted by the components `value` prop.
 -const isValid = adapterDateFns.isValid('2022-04-17');
 +const isValid = adapterDateFns.isValid(new Date('2022-04-17'));
 ```
+
+### Extend `dayjs` in `AdapterDayjs` constructor
```

Comment:
I would move it outside of the adapter methods listing because those sections don't impact people that do not instantiate manually the adapter whereas your new one can impact people that do not do it.

How about we move `Use UTC with the Day.js adapter` to be an H3 with a new H2 `Usage with Day.js` that would include both?